pallets/automation-time/src/fees.rs

```diff
@@ -19,6 +19,7 @@
 use crate::{AccountOf, Action, ActionOf, Config, Error, MultiBalanceOf, Pallet};
 
 use frame_support::traits::Get;
+use frame_system::RawOrigin;
```

Comment:
Why is the addition of this line? Is it necessary?

Reply:
Yes.
During testing, we require Root permissions to deposit funds into the test account.